test/acceptance/go.mod

```diff
@@ -11,3 +11,46 @@ require (
 	github.com/hashicorp/consul/sdk v0.9.0
 	github.com/stretchr/testify v1.4.0
 )
+
+require (
```

Comment:
I guess this is just due to the version bump to go 1.19

Reply:
Yeah, it looks like [in 1.17](https://go.dev/doc/go1.17#go-command) they started to include indirect dependencies in go.mod:

> Because the number of explicit requirements may be substantially larger in an expanded Go 1.17 go.mod file, the newly-added requirements on indirect dependencies in a go 1.17 module are maintained in a separate require block from the block containing direct dependencies.